### PR TITLE
Fixed Rakefile to call appraisal when you type "rake" on the command line

### PR DESCRIPTION
**Pull request by jasnow**
* Fixed Rakefile to call appraisal when you type "rake" on the command line.
 * Upgraded .travis.yml to 2.2.4 and 2.1.8 and 2.3.0.
 *  Moved .rvmrc and .overcommit.yml lines to global .gitignore file and removed it from local .gitignore file.

### CODE REVIEW

**Comment by jasnow** [comment]
Green Travis build: https://travis-ci.org/jasnow/setler/builds/105775990

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,3 @@ coverage.data
 .bundle
 Gemfile.lock
 pkg/*
-# Added so not to force others to use .rvmrc and .overcommit.yml files.
-.rvmrc
-.overcommit.yml
```
Comment (ckdake) [inline .gitignore]: Why pull these out now?

Comment (jasnow) [inline .gitignore]: Some projects that I contribute to did not want them so I added them to my global .gitignore file.
Similar to you not wanting my Gemfile.lock file - some projects want them and some don't.

Comment (ckdake) [inline .gitignore]: :+1: less is more :)

**Comment by jasnow** [comment]
Ping me if there is a problem with this. Cheers.

**Pull request merged**